.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: python
+dist: trusty
+
+python:
+  - '3.6'
+  - 'nightly'
+
+sudo: false
+
+cache: pip
+
+install:
+  - pip install --requirement requirements.txt
+  - pip install pylint
+
+script:
+  - pylint mallard
+
+notifications:
+  email:
+    on_success: change
+    on_failure: always
```

Comment:
can we use something like this instead? https://github.com/Techtony96/TravisCI-Webhooks/ i'm not a fan of getting spammed with mails, but I'm not entirely sure when and to whom travis mail notifications send anyways

Reply:
We removed the webhooks because they were too spammy. With these settings you only get emails if the build fails, not every single time you push. I've used it on other projects and I like this balance.

But I don't think you get any of the emails anyways,